templates/company/officers/list.html.tx

```diff
@@ -118,7 +118,11 @@
                     <dl class="column-quarter">
                         <dt><% l('Role') %>
                             % if $officer.resigned_on  {
-                               <span id="officer-status-tag-<% $~officer.count %>" class="status-tag ceased-tag font-xsmall">Resigned</span>
+                                % if $officer.identification.identification_type == 'registered-overseas-entity-corporate-managing-officer' {
```

Comment:
We may need to check for non corporate managing officers also. Just waiting to find out.

Reply:
Yes this will not cover both types of managing officer. Will need to be updated for both corporate and non corporate MO's.